ksmt-bitwuzla/src/main/kotlin/org/ksmt/solver/bitwuzla/KBitwuzlaModel.kt

```diff
@@ -185,4 +189,18 @@ open class KBitwuzlaModel(
         if (other !is KModel) return false
         return detach() == other
     }
+
+    /**
+     * Models are currently not supported for formulas with quantifiers.
+     * Handle this case as unsupported feature rather than general solver failure.
+     * */
+    private inline fun <T> handleModelIsUnsupportedWithQuantifiers(body: () -> T): T = try {
+        body()
+    } catch (ex: BitwuzlaNativeException) {
+        val modelUnsupportedMessage = "'get-value' is currently not supported with quantifiers"
```

Comment:
Looks quite odd. Can this string be extracted into a constant or replace it with something less expensive? At least, `startsWith` instead of `in`